ledger-core/application/preprocessor/templates/wrapper.go.tpl

```diff
@@ -138,8 +138,11 @@ func INSMETHOD_{{ $method.Name }}(object []byte, data []byte) (newState []byte,
 		if !needRecover {
 			return
 		}
-		if recoveredError := throw.RW(recover(), nil, "Failed to execute method (panic)"); recoveredError != nil {
-			recoveredError = ph.MakeErrorSerializable(recoveredError)
+		if r := recover(); r != nil {
```

Comment:
I strongly recommend use of Goexit() as it can't be recovered by recover()

will push a patch

Reply:
but this can be an overkill now

Reply:
redo to Goexit()